lxd/storage/backend_lxd.go

```diff
@@ -3466,7 +3466,8 @@ func (b *lxdBackend) CreateBucket(projectName string, bucket api.StorageBucketsP
 
 	revert.Add(func() { _ = BucketDBDelete(context.TODO(), b, bucketID) })
 
-	storageBucket := b.GetVolume(drivers.VolumeTypeBucket, drivers.ContentTypeFS, bucket.Name, bucket.Config)
+	storageBucketName := project.StorageVolume(projectName, bucket.Name)
+	storageBucket := b.GetVolume(drivers.VolumeTypeBucket, drivers.ContentTypeFS, storageBucketName, bucket.Config)
```

Comment:
Can we keep storageBucket called vol or bucketVol please, as storageBucket doesnt tell the reader whether it represents the bucket itself or the volume serving it.

Reply:
I realise it was storageBucket before, but think we could make things clearer by standardising on bucketVol :)

Reply:
I agree, `bucketVol` is easier to understand